opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -926,7 +926,7 @@
   "Enable 3D mode": "Activer le mode 3D",
   "Enable AI powered platform": "Activer la plateforme alimentée par l'IA",
   "Enable commit messages": "Activer les messages de justification",
-  "Enable Enterprise Edition": "Enable Enterprise Edition",
+  "Enable Enterprise Edition": "Activer l'édition Enterprise",
```

Comment:
Would be better to put the same name for Enterprise Edition everywhere, @nino-filigran or @romain-filigran do you prefer "édition Entreprise", "Edition Entreprise" or "Enterprise Edition" even in French ?